lab_5_scrapper/scrapper.py

```diff
@@ -168,6 +290,11 @@ def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         Args:
             article_soup (bs4.BeautifulSoup): BeautifulSoup instance
         """
+        text = article_soup.find_all(style="text-align: justify;")
```

Comment:
looks like too broad condition - you will get some more data than you need

Reply:
i tried it and got a text only from an article

Reply:
the only thing is that i get some information about a source of a photo, but it is present in articles and located in articleBody, should i get rid of it?